Library/Tests/TestHelpers/TestCase.swift

```diff
@@ -15,8 +16,9 @@ internal class TestCase: FBSnapshotTestCase {
   internal let cookieStorage = MockCookieStorage()
   internal let dateType = MockDate.self
   internal let facebookAppDelegate = MockFacebookAppDelegate()
+  internal let liveStreamService = MockLiveStreamService(fetchEventResult: nil)
   internal let mainBundle = MockBundle()
-  internal let scheduler = TestScheduler()
+  internal let scheduler = TestScheduler(startDate: MockDate().date)
```

Comment:
we did some fun new time-based testing stuff in this PR, and this lil fella was the key to having our test scheduler and mock date stuff in sync. i'll call out some other cool stuff we did later...